### PR TITLE
New Dashboard [#155689482]

### PR DESCRIPTION
**Pull request by thematthopkins**
Some dependencies for the impact-app + management-app version PR.  The main change here is including the redirector + downloader related services, so they can be shared between the two apps.

### CODE REVIEW

File: addon/components/impact-circle.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ export default Component.extend({
       const circle = this.element;
       let hasLargeClass = false;
       for (let i = 0; i < circle.classList.length; i++) {
-        if (circle.classList[i] === 'large') {
+        if (circle.classList[i] === 'large' || circle.classList[i] === 'large') {
```
Comment (jonpitch) [inline addon/components/impact-circle.js]: should this be `xlarge`?

File: addon/models/assessment.js [hunk 2]
```diff
@@ -120,6 +120,12 @@ export default Model.extend({
   createWorkingCopy: memberAction({
     path: 'working-copy',
     type: 'post'
+  }),
+
+  scoreSection: computed('sections', function(){
+    return get(this, 'sections').filter(s => {
+      s.slug == 'b-impact-report';
```
Comment (jonpitch) [inline addon/models/assessment.js]: nit - `===`

File: .eslintrc.js [hunk 3]
```diff
@@ -16,10 +16,7 @@ module.exports = {
   },
   rules: {
     'ember/no-observers': 2,
-    'ember/no-jquery': 1,
-    'ember/named-functions-in-promises': [2, {
```
Comment (jonpitch) [inline .eslintrc.js]: didn't care for this lint rule, eh?

File: addon/styles/impact/components/circle.scss [hunk 4]
```diff
@@ -85,6 +85,25 @@ impact-circle.large {
   }
 }
 
+impact-circle.xlarge {
+  div {
+    @include circle-size(13.5em);
```
Comment (jonpitch) [inline addon/styles/impact/components/circle.scss]: i can't wait to see this mythical extra large circle

File: addon/components/impact-circle.js (hunk 1 above)
Comment (thematthopkins) [reply addon/components/impact-circle.js]: Thanks eagle-eyes !

**Pull request merged**